### PR TITLE
Upgrade to hapi 17

### PR DESCRIPTION
**Pull request by dawnerd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,17 +27,17 @@
...
-    "eslint": "^3.9.1",
-    "eslint-config-firstandthird": "^3.2.0",
+    "eslint": "^4.13.1",
+    "eslint-config-firstandthird": "^4.3.0",
     "eslint-plugin-import": "^2.1.0",
-    "hapi": "^16.1.1",
-    "hapi-logr": "^3.0.0",
-    "logr-flat": "^2.0.0"
+    "hapi": "^17.2.0",
+    "hapi-logr": "^4.0.0",
```
Comment (jgallen23) [inline package.json:40]: can remove these now

File: package.json [hunk 2]
```diff
@@ -24,17 +27,17 @@
   },
   "homepage": "https://github.com/firstandthird/hapi-oppsy#readme",
   "dependencies": {
-    "oppsy": "1.0.2"
+    "oppsy": "2.0.0"
```
Comment (jgallen23) [inline package.json]: maybe ^2.0.0?

**Review by jgallen23 — CHANGES_REQUESTED** [review]

File: example/index.js [hunk 3]
```diff
@@ -27,38 +26,48 @@ server.register([
...
     }
   });
+
   server.route({
     path: '/redirect',
     method: 'GET',
-    handler(req, reply) {
-      reply.redirect('https://google.com');
+    handler(req, h) {
+      h.redirect('https://google.com');
```
Comment (jgallen23) [inline example/index.js]: need to return

**Review by jgallen23 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (dawnerd) [reply package.json:40]: ~~The example script uses it~~ nvm I see

Edit: Actually removing breaks example after all

**Pull request merged**